.pipelines/templates/PackageBuild.yml

```diff
@@ -185,6 +187,7 @@ steps:
       fi
 
       sudo make -C "${{ parameters.buildRepoRoot }}/toolkit" build-packages -j$(nproc) \
+        TOOLCHAIN_ARCHIVE="$(toolchainArchive)" \
```

Comment:
If `parameters.customToolchainArtifactName` is false, then `toolchainArchive` won't be set to anything. If the pipeline can't find a variable, then it doesn't modify the string. So, `TOOLCHAIN_ARCHIVE` would be set to literally `$(toolchainArchive)`, which would probably confuse `make`.

Reply:
You're right, I completely missed that and though non-set pipeline variables are evaluated to an empty string. @dmcilvaney, in that case adding a `- ${{ else }}:` part, where we set `toolchainArchive` to an empty string should help.

Reply:
Alternatively, I'd also support doing something similar we do for a few other arguments:
```
if [[ -n ${{ parameters.customToolchainArtifactName }} == "true" ]]; then
        toolchain_archive_arg="TOOLCHAIN_ARCHIVE=$(toolchainArchive)"
fi
(...)
sudo make -C "${{ parameters.buildRepoRoot }}/toolkit" build-packages -j$(nproc) \
(...)
$run_check_arg \
$toolchain_archive_arg \
```

Reply:
Ok, will try the arg variable since that is the pattern we are already using.